metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -542,6 +545,50 @@ object Messages {
     }
   }
 
+  object UpdateScalafmtConf {
+
+    def letUpdate = new MessageActionItem("Let Metals update .scalafmt.conf")
+    def createMessage(
+        sourceItems: List[RelativePath],
+        dialect: ScalafmtDialect
+    ): String = {
+      val directories = sourceItems.mkString("[", ",\n", "]")
```

Comment:
Can we just print the number, but log everything and just mention that? It might actually be a lot of directories sometimes.

Reply:
I think it's better to show these directories because there are several options for solving this warning.
In addition to automatic rewrite, you might take these paths and do `fileOverride` by hand or add them in `excludePaths`.

Also, there is yet another one - create an issue that Metals wrongly detects dialect :smile:

Reply:
How will the message look in case of a really large project? In case you have 20 modules with test and main sources it will print 40 directories. You can say `look in the logs to find out which.` And you can even use the same logs link that we use when importing the build.

Reply:
Moved `directories` list to logs